quickstart.rb

```diff
@@ -53,4 +58,14 @@ def init_service
 end
 
 contact_fixer = ContactFixer.new(init_service, $stdout)
-contact_fixer.print_connections(contact_fixer.get_all_contacts)
+all_contacts = contact_fixer.get_all_contacts
+cli = HighLine.new
+
+
+first_filter = cli.ask("What filter do you want ot run?  ") { |q| q.default = ".*" }
```

Comment:
If first_filter is going to be used regex for the replacement step, then we should replace the current default value with "\+.\*" (in the replacement process the '.*' expression treats '+' as a separated unit).